app/models/calendar_event.rb

```diff
@@ -87,11 +84,11 @@ def self.project_date_filter(range, scope = Project)
   end
 
   def self.project_step_date_filter(range, scope = ProjectStep, project_id)
-    scoped_steps = scope.where("actual_end_date BETWEEN :first AND :last "\
```

Comment:
We changed which events we want to show up. Any events that occur over the range, even if they start before or end after the range, should be displayed.